src/locales/en-us/common/index.yaml

```diff
@@ -19,6 +19,9 @@ common:
     title: 'An error has occurred while trying to load this data.'
     details: 'Details'
     field: 'Field'
+  delete_modal:
+    api_error: 'Error {status}: {title}'
+    default_error: 'An unexpected error occurred'
```

Comment:
Why does `delete_modal` has specifically defined error messages, surely we want to define this sort of stuff in one place so its consistent?

Reply:
Not sure I’m following. What do you mean by this?

Note that this object now also contains the messages from https://github.com/kumahq/kuma-gui/pull/1521#discussion_r1331569390.

Reply:
Well aren't these generic/common error messages/text? Or do we only use them in the delete modal?

Reply:
These are currently only used in the delete modal. They _are_ pretty generic and could be used elsewhere.